packages/interactivity/src/proxies/index.ts

```diff
@@ -0,0 +1,5 @@
+/**
+ * Internal dependencies
+ */
+export { proxifyState, peek } from './state';
+export { proxifyStore } from './store';
```

Comment:
Not sure about the naming here. I feel "proxify" is comprehensible, but maybe the appropriate word is "proxy", used as a verb.

What do you think? 🤔

Reply:
I also think `proxify` is sounds all right 🙂